README.md

```diff
@@ -1,2 +1,77 @@
 # Sonarqube
 
+# Table of contents
+1. [Introduction](#what-is-sonarqube)
+2. [Requirement](#requirement)
+3. [Usage](#usage)
+
+## Introduction
+[**Sonarqube**](https://www.sonarqube.org/)  is the leading tool for continuously inspecting the Code Quality and Security of your codebases, and guiding development teams during Code Reviews. 
+
+## Requirement
+- Installing [**Docker**](https://www.docker.com/)
+
+## Usage
+
+1. Installing SonarQube locally on Linux by entering the following commands in to terminals:
+
+    1. Pull Sonarqube image 
+    ```
+    docker pull sonarqube
+    ```
+
+    2. Start the server by running:
+    ```
+    docker run -d --name sonarqube -e SONAR_ES_BOOTSTRAP_CHECKS_DISABLE=true -p 9000:9000 sonarqube:latest
+    ```
+
+    >Please waiting a few minutes, Log in to http://localhost:9000 using System Administrator credentials:
+    >- login: admin
+    >- password: admin
+
+    </br>
+
+    ![Login-page](assets/login-page.jpg)
```

Comment:
Naming convetion for this file is not suitable in linix system, Please using underscore